platform/openide.util.ui/src/org/openide/util/FilteredIcon.java

```diff
@@ -87,4 +89,8 @@ protected Image createAndPaintImage(
         return Toolkit.getDefaultToolkit().createImage(
                 new FilteredImageSource(img.getSource(), filter));
     }
+    
+    public String getFilterType() {
+        return filterType;
```

Comment:
This is a leftover from an additional feature I am not able to deliver atm because of time constraints; will follow up in another PR. There should be something as 'image composition' for images merged by `ImageUtiltiies` - now after adjustments that would be represented as a String[]. That would allow clients to really distinguish bare and decorated images, should they need so (the LSP client may need, since the badges change semantic that may be represented differently by the LSP client).

Removed in 6c944b9